helm/istio/charts/istio_cni/templates/serviceaccount.yaml

```diff
@@ -1,7 +1,5 @@
-{{ if .Values.enabled }}
```

Comment:
See my comment about setting this in the controller.  @luksa is doing something similar to hard code the namespace used for the cni resources (moving them into operator namespace).

Also, removing all these diff's to the templates will save @luksa from having to resolve conflicts in his PR.